pages/_language/events/_event/index.vue

```diff
@@ -1,41 +1,100 @@
 <template>
-  <main class="event">
-    <h2 class="event__title">{{ event.title }}</h2>
-    <nav>
-      <ul>
-        <li v-for="{ locale } in event._allTitleLocales" :key="locale">
-          <nuxt-link
-            rel="alternate"
-            :hreflang="locale"
-            :lang="locale"
-            :to="`/${locale}/events/${params.event}`"
-          >
-            {{ locale }}
-          </nuxt-link>
-        </li>
-      </ul>
-    </nav>
-
-    <ul class="event__relevant-chapters">
-      <li
-        class="chapter-preview"
-        v-for="chapter in event.relevantChapters"
-        :key="chapter.slug"
+  <div class="event">
+    <header class="event-header">
+      <div class="event-header__intro">
+        <h1 class="event__name">
+          <span class="event__location">{{ event.location }}</span>
+          {{ event.name }}
+        </h1>
+        <nav class="language-switch">
+          <ul class="language-switch__list">
+            <li
+              v-for="{ locale } in event._allNameLocales"
+              :key="locale"
+            >
+              <nuxt-link
+                class="language-switch__link"
+                :class="{ 'language-switch__link--active': params.language === locale }"
+                rel="alternate"
+                :hreflang="locale"
+                :lang="locale"
+                :to="`/${locale}/events/${params.event}`"
+              >
+                {{ locale }}
```

Comment:
The locale is presented here as a language code, but is it maybe possible to display it in the language of the user? If not, the `lang`-attribute is not necessary, as it is in the same language as the page language.

Reply:
Yes definitely, will need to look into a language mapping perhaps using one of these:
    https://www.npmjs.com/package/locale-code
    https://www.npmjs.com/package/country-locale-map
    https://www.npmjs.com/package/iso-639-1

Think this task is big enough for a new ticket.